index.d.ts

```diff
@@ -0,0 +1,45 @@
+interface Authentication {
+	username: string;
+	password?: string;
+}
+
+interface Options {
+	width?: number;
+	height?: number;
+	type?: 'png' | 'jpeg';
+	quality?: number;
+	scaleFactor?: number;
+	emulateDevice?: string;
+	fullPage?: boolean;
+	defaultBackground?: boolean;
+	timeout?: number;
+	delay?: number;
+	waitForElement?: string;
+	element?: string;
+	hideElements?: string[];
+	removeElements?: string[];
+	clickElement?: string;
+	modules?: string[];
+	scripts?: string[];
+	styles?: string[];
+	headers?: object;
+	userAgent?: string;
+	cookies?: (string | object)[];
+	authentication?: Authentication;
+	beforeScreenshot?: Function;
+	debug?: boolean;
+	launchOptions?: object;
+	overwrite?: boolean;
+}
+
+export const devices: string[];
+
+export function base64(url: string, options?: Options): Promise<string>;
+
+export function buffer(url: string, options?: Options): Promise<Buffer>;
+
+export function file(
+	url: string,
+	filePath: string,
```

Comment:
I know it's like this in the readme too, but I think it would be clearer like this:

```suggestion
	outputFilePath: string,
```